### ISSUE TITLE
Un-natural shoulder movement.

### ISSUE THREAD

**Issue by rohaantahir** [zhangchenxu528/FACIAL#74]
Hi, I have been working on you project for video synthesis. I have successfully been able to generate the video with good quality lip syncing. But the output video seems to have some un-natural shoulder movements can you please suggest what should I do to keep them as it is in the input video.

**Comment by rostekus** [comment]
Off-topic
Hi, can you share your video resault? My email rostyslavmosorov@gmail.com

**Comment by rohaantahir** [comment]
Sent you the email kindly check it. thanks

**Comment by rohaantahir** [comment]
@rostekus have you checked??

**Comment by rostekus** [comment]
@rohaantahir  Yes. I have, I'e been playing with repo for a while and I think your problem is due to different body position then in Obama case. So it is very likely that the network has a problem with proper generating lower body parts

**Comment by rohaantahir** [comment]
Thanks for the kind reply. Is it possible to overcome this issue in near feature? are you working on it ??

**Comment by rohaantahir** [comment]
@rostekus any update kindly?

**Comment by rostekus** [comment]
@rohaantahir Sorry, but I'm not the author of this repo and I don't work on any issue
